examples/models/image_classification/LeNet5.py

```diff
@@ -0,0 +1,176 @@
+
+        # Compute train accuracy
+        (train_loss, train_acc) = self._model.evaluate(X_validation, y_validation)
+
+    def evaluate (self, dataset_uri):
+        dataset = dataset_utils.load_dataset_of_image_files(dataset_uri, image_size=[28,28])
+        (images, classes) = zip(*[(image, image_class) for (image, image_class) in dataset])
+        images = self._prepare_X(images)
+        images = np.pad(images, ((0,0),(2,2),(2,2),(0,0)), 'constant')
+
+        X_test, y_test = images, to_categorical(classes, num_classes = 10)
+
+        # Compute test accuracy
+        (test_loss, test_acc) = self._model.evaluate(X_test, y_test)
+        return test_acc
+    
+    def predict(self, queries):
+        X = self._prepare_X(queries)
+        X = np.pad(X, ((0,0),(2,2),(2,2),(0,0)), 'constant')
+        probs = self._model.predict_proba(X)
+        return probs.tolist()
+        
+    def destroy(self):
+        pass
+
+    def dump_parameters(self):
+        params = {}
+
+        # Save model parameters
+        model_bytes = pickle.dumps(self._model)
+        model_base64 = base64.b64encode(model_bytes).decode('utf-8')
+        params['model_base64'] = model_base64
+        
+        return params
+
+    def load_parameters(self, params):
+        # Load model parameters
+        model_base64 = params.get('model_base64', None)
+        if model_base64 is None:
+            raise InvalidModelParamsException()
+        
+        model_bytes = base64.b64decode(params['model_base64'].encode('utf-8'))
+        self._model = pickle.loads(model_bytes)
+    def _prepare_X(self, images):
+        X = np.asarray(images)
+        return X.reshape(-1,28,28,1)
+            
+    def _build_classifier(self, epochs, batch_size):
+        model = models.Sequential()
+        model.add(layers.Conv2D(filters=6, kernel_size=(3, 3), activation='relu', input_shape=(32,32,1)))
+        model.add(layers.AveragePooling2D())
+        model.add(layers.Conv2D(filters=16, kernel_size=(3, 3), activation='relu'))
+        model.add(layers.AveragePooling2D())
+        model.add(layers.Flatten())
+        model.add(layers.Dense(units=120, activation='relu'))
+        model.add(layers.Dense(units=84, activation='relu'))
+        model.add(layers.Dense(units=10, activation = 'softmax'))
+        
+        adam = Adam(lr=0.001, beta_1=0.9, beta_2=0.999, epsilon=None, decay=0.0, amsgrad=False)
+
+        model.compile(loss=keras.losses.categorical_crossentropy, optimizer=adam, metrics=['accuracy'])
+        return model
+
+if __name__ == '__main__':
+    test_model_class(
+        model_file_path=__file__,
+        model_class='LeNet5',
+        task=TaskType.IMAGE_CLASSIFICATION,
+        dependencies={
+            ModelDependency.TENSORFLOW: '1.12.0',
+            ModelDependency.KERAS: '2.2.4'
+        },
+    train_dataset_uri='data/fashion_mnist_for_image_classification_train.zip',
+    test_dataset_uri='data/fashion_mnist_for_image_classification_test.zip',
```

Comment:
These parameters no longer works on `dev`. Need to update then. Refer to sample models on `dev` e.g. https://github.com/nginyc/rafiki/blob/dev/examples/models/image_classification/SkDt.py